### PR TITLE
Release prep

### LINKED ISSUE #1: CLTS errors
@SimonGreenhill Thanks for transferring the repo!

We currently have 45 CLTS errors which we'd need to fix. Many of them are diacritics whose convention I am not familiar with in the Oceanic area. Mostly `^` and `ṣ` (syllabic?. There are also some cases in brackets, like the following:

| ID | LANGUAGE | CONCEPT | FORM | SEGMENTS |
|:--------------------------|-----------:|:--------------------|:---------------|:----------------------------------------------------------|
| 13-194_how-2 | 13 | 194_how | (k)abitáy | <s> <<(>> </s> k <s> <<)>> </s> a b i t á/a y/j |
| 13-207_twenty-1 | 13 | 207_twenty | duwá (ka) púlu | d u w á/a + <s> <<(>> </s> k a <s> <<)>> </s> + p ú/u l u |
| 13-208_fifty-1 | 13 | 208_fifty | líma (ka) púlu | l í/i m a + <s> <<(>> </s> k a <s> <<)>> </s> + p ú/u l u |

If you can give me some answers to those cases, I can try to implement the CLTS fixes myself.

### LINKED ISSUE #2: Contributors
@SimonGreenhill Are you the only contributor to this dataset or is it the same team as abvdoceanic? Should we cite all the authors form the 2008 paper as authors, in addition to the 2024 authors?

### PR DESCRIPTION
**Pull request by FredericBlum**
@SimonGreenhill Two major changes that I would need you to review:

- Many additions to the orthography profile. Please confirm or comment on the changes if I have introduced any errors. Mostly educated guesses, following your comments in the issue (closes #1 )
- Added all the authors of the 2008 and 2024 paper as authors. Any other Contributors that should be added? (closes #2 )

Once we have fixed this, I will prepare the Zenodo release (v1.0 or v0.1?)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,7 +14,7 @@ If you use these data please cite
 
 This dataset is licensed under a CC-BY-4.0 license
 
-Available online at https://abvd.eva.mpg.de/austronesian/
+Available online at https://abvd.shh.mpg.de/austronesian/
```
Comment (SimonGreenhill) [inline README.md]: URL here should be .eva. not .shh.

File: README.md [hunk 2]
```diff
@@ -74,10 +72,10 @@ To make a nexus file, use the custom `abvdphilippines.nexus` in cldfbench. The p
 
 ```shell
 # make a nexus file, with combined cognates removed above level 2:
-cldfbench abvdphilippines.nexus --removecombined 2 --output abvd_philippines.nex
+cldfbench abvdphilippines.nexus --removecombined 2 --output abvdoceanic.nex
```
Comment (SimonGreenhill) [inline README.md:73]: can we change the outfile here to abvd_philippines.nex?

File: README.md [hunk 3]
```diff
@@ -74,10 +72,10 @@ To make a nexus file, use the custom `abvdphilippines.nexus` in cldfbench. The p
...
 
 # ...with per-word ascertainment correction:
-cldfbench abvdphilippines.nexus --ascertainment=word --removecombined 2 --output abvd_philippines.nex
+cldfbench abvdphilippines.nexus --ascertainment=word --removecombined 2 --output abvdoceanic.nex
 ````
```
Comment (SimonGreenhill) [inline README.md]: and here too.

**Review by SimonGreenhill — COMMENTED** [review]
all looks good except for these three minor things, thanks!

**Comment by FredericBlum** [comment]
Thanks! I have fixed the remaining issues and will merge/release now.

**Pull request merged**